src/Select/Select.jsx

```diff
@@ -105,6 +111,8 @@ Select.propTypes = {
   placeholder: React.PropTypes.string,
   readOnly: React.PropTypes.bool,
   searchable: React.PropTypes.bool,
+  creatable: React.PropTypes.bool,
+  creatableProps: React.PropTypes.object,
```

Comment:
yeah, we shouldn't need a separate component. hoping we can just copy the list of proptypes over from the `Creatable` and list them here instead of requiring users to look them up in react-select docs. interface for a normal select and a create-enabled one would be the same; the user can just include the additional props.
```jsx
<Select
  options={[...]}
  creatable
  newOptionCreator={() => {...}}
  isValidNewOption={() => {...}}
/>
```